src/components/subscriptions/licenses/LicenseAllocationHeader.jsx

```diff
@@ -1,23 +1,28 @@
 import React, { useContext } from 'react';
 import { SubscriptionDetailContext } from '../SubscriptionDetailContextProvider';
-import LicenseAllocationBanner from './LicenseAllocationBanner';
+import { SubsidyRequestConfigurationContext } from '../../subsidy-request-configuration';
 import NewFeatureAlertBrowseAndRequest from './NewFeatureAlertBrowseAndRequest';
+import { SUPPORTED_SUBSIDY_TYPES } from '../../../data/constants/subsidyRequests';
 
 const LicenseAllocationHeader = () => {
   const {
     subscription,
   } = useContext(SubscriptionDetailContext);
+  const { subsidyRequestConfiguration } = useContext(SubsidyRequestConfigurationContext);
+
+  // don't show alert if the enterprise already has subsidy requests enabled
+  const isBrowseAndRequestFeatureAlertShown = subsidyRequestConfiguration?.subsidyType
```

Comment:
[clarification] What if someone has not yet visited the "Settings" page yet for a customer? Is `isBrowseAndRequestFeatureAlertShown` be `falsey` since `subsidyRequestConfiguration` would be null/undefined in this case? If so, would we still want to direct admins to the "Settings" page if a `subsidyRequestConfiguration` has not yet been created?

Reply:
subsidyRequestConfiguration is guaranteed to have been created by this point if the enterprise is eligible. So the only way, if it's null, is if the enterprise has both subsidies or if an error occurred in between.

Reply:
Ignore this. `subsidyRequestConfiguration` is guaranteed to be created at this point, under the right circumstances.